### PR TITLE
Threat Intel section

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
Adds a section on threat intelligence to the existing Security Signals Explorer docs

### Motivation
Jira request

### Preview
https://docs-staging.datadoghq.com/sarina/threat-intel-docs/security_platform/explorer

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security_platform/explorer.md [hunk 1]
```diff
@@ -57,6 +57,21 @@ Below the overview of the signal are tabs with detailed information related to t
 
 - `Related Issues` includes a list of other signals which contain the same group by values to assist with triaging the signal.
 
+### Threat Intelligence
+
+Datadog Security Monitoring offers threat intelligence feeds curated by threat intelligence partners. These feeds are constantly updated to include data about known suspicious activity (for example, IOCs), so you can quickly identify which potential threats to address.
+
+{{< img src="security_platform/security_monitoring/explorer/threat_intelligence.png" alt="Threat Intelligence in the Security Signals Explorer"  >}}
+
+Datadog automatically operationalizes threat intelligence by analyzing all ingested logs that have relevant attributes. If there is an indicator of compromise in a log, such as an anonymized IP tied to a VPN, proxy, or Tor exit node, a `threat_intel` attribute is append to the log event to provide additional insights based on available intelligence.
```
Comment (kayayarai) [inline content/en/security_platform/explorer.md:66]:
```suggestion
Datadog automatically operationalizes threat intelligence by analyzing all ingested logs that have relevant attributes. If a log contains a compromise indication, such as an anonymized IP tied to a VPN, proxy, or Tor exit node, a `threat_intel` attribute is append to the log event to provide additional insights based on available intelligence.
```

File: content/en/security_platform/explorer.md [hunk 2]
```diff
@@ -57,6 +57,21 @@ Below the overview of the signal are tabs with detailed information related to t
...
+
+The query to see all threat intel matches in the Security Signals Explorer is `@threat_intel.indicators_matched:*`. The following are additional attributes to query for threat intelligence:
+
+* `@threat_intel.results.category “anonymizer”, “scanner”`
+* `@threat_intel.results.intention “malicious”, “unknown”`
+* `@threat_intel.results.subcategory options "proxy", "tor", "vpn"`
+    **Note**: These are only provided by IPinfo.
```
Comment (kayayarai) [inline content/en/security_platform/explorer.md:73]:
```suggestion
    **Note**: Proxy, Tor, and VPN subcategory attributes are provided only by threat intelligence partner IPinfo.
```
I think that's what you mean?

File: content/en/security_platform/explorer.md [hunk 3]
```diff
@@ -57,6 +57,21 @@ Below the overview of the signal are tabs with detailed information related to t
 
 - `Related Issues` includes a list of other signals which contain the same group by values to assist with triaging the signal.
 
+### Threat Intelligence
+
+Datadog Security Monitoring offers threat intelligence feeds curated by threat intelligence partners. These feeds are constantly updated to include data about known suspicious activity (for example, IOCs), so you can quickly identify which potential threats to address.
+
+{{< img src="security_platform/security_monitoring/explorer/threat_intelligence.png" alt="Threat Intelligence in the Security Signals Explorer"  >}}
```
Comment (kayayarai) [inline content/en/security_platform/explorer.md:64]: This image looks like it needs some blurring or cropping of sensitive info

**Review by kayayarai — CHANGES_REQUESTED** [review]
Left you some comments!

File: content/en/security_platform/explorer.md (hunk 3 above)
Comment (sarina-dd) [reply content/en/security_platform/explorer.md:64]: @jmassey-datadog Does this image contain sensitive info? It's being used in other places throughout the docs / blog.

**Review by kayayarai — APPROVED** [review]

**Pull request merged**